modules/oe_theme_helper/oe_theme_helper.post_update.php

```diff
@@ -296,3 +296,23 @@ function oe_theme_helper_post_update_20016() {
     'label' => 'OpenEuropa: Compact teaser',
   ])->save();
 }
+
+/**
+ * Update default view display for Text with featured media paragraph.
+ */
+function oe_theme_helper_post_update_20017() {
```

Comment:
This is now ran by oe_paragraphs, we should remove it from here. Same goes for the configuration file itself.